### ISSUE TITLE
Prediction.DisableHostEvents() to complement the new RPC History System

### ISSUE THREAD

**Issue by veesusmikelheir** [Facepunch/sbox-issues#628]
**What can't you do?**
Disable all RPC's for a predicting client, mainly for RPC's where it doesn't make a practical difference if they have same params on server or client, or where its impossible to guarantee they'll be exactly the same on server or client

**How would you like it to work?**
Add a new Prediction disposable like Prediction.Off(), except instead of explicitly preventing rpc culling on the predicting client, it explicitly forces RPC culling. This new method could be called Prediction.DisableHostEvents or Suppress or something similar.
In short, it would allow developers to opt in to the old system of RPC culling before the history system was implemented.

**What have you tried?**
Seeding RPC parameters and using parameterless RPC's. These can still get past the RPC history system due to quirks in networking or slight differences in entity states on client and server causing them to not compare with anything in the RPC History (see #600)

**Additional context**
#600

**Comment by garrynewman** [comment]
Wouldn't it be better to fix the issues with it?
